tools/memory_estimator/memory_estimator.py

```diff
@@ -52,6 +63,63 @@
 }
 
 
+def apply_patches(freertos_lts, lib_name):
```

Comment:
can we add a comment here on why we are doing this.

Reply:
Done.